siphon/__init__.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2013-2015 University Corporation for Atmospheric Research/Unidata.
 # Distributed under the terms of the MIT License.
 # SPDX-License-Identifier: MIT
+"""Tools for accessing atmospheric and oceanic science data on remote servers."""
```

Comment:
The only TDSs I've encountered in the wild are AOS. I'd say it's ok for now, especially since it is kind of the target audience, and siphon is still pretty unknown (I think).